frontend/src/queries/nodes/InsightViz/EditorFilterGroup.tsx

```diff
@@ -47,6 +47,11 @@ export function EditorFilterGroup({
             {isRowExpanded ? (
                 <div className="EditorFilterGroup__content">
                     {editorFilters.map(({ label: Label, tooltip, showOptional, key, component: Component }) => {
+                        if (Component && Component.name === 'component') {
+                            throw new Error(
```

Comment:
Probably should be in `frontend/src/scenes/insights/EditorFilters/EditorFilterGroup.tsx` as well. Otherwise only devs with the feature flag activated see the warning.